aurora-multiphysics/hippo
# Make updates for compatibility with latest MOOSE version

## Summary


To use Hippo with the latest version of MOOSE we had to:

- Update `bool FoamProblem::converged()` to the new form required by MOOSE `bool FoamProblem::converged(const unsigned int)`.
- Update tests that used `RunCommand`, as upstream changes meant that the given command was no longer passed directly to the shell. It seems that the given string's first argument is taken as the executable and all subsequent arguments are passed to it, rather than the string being treated as a shell command. The fix here was to prefix all the command strings with `bash`.
- Update the CI Docker image to use the latest version of MOOSE. This image is based off the `moose-dev` image, which runs Rocky Linux rather than Ubuntu.

## Related Issue

Resolves #32 

## Checklist

- [ ] Tests have been written for the new/changed behaviour.
- [ ] Documentation/examples have been added/updated for the new changes.
